examples/md-flexible/CMakeLists.txt

```diff
@@ -92,10 +95,55 @@ add_test(
         CONFIGURATIONS checkExamples
 )
 
+set(HAS_AVX true)
```

Comment:
Write a short justification comment for this whole code block. Would be nice to know what is coming without reading 50 lines